### PR TITLE
Fix 1974 build page TypeError

### LINKED ISSUE #1974: Builds page may throw a TypeError if object not found.
url: 
`https://localhost:9000/dev-console/project/myproject/browse/builds`

Error:
screen shot 2017-08-28 at 11 50 46 am

Problem line `66`
screen shot 2017-08-28 at 11 51 40 am

### PR DESCRIPTION
**Pull request by benjaminapetersen**
fixes #1974

@spadgett

### CODE REVIEW

File: app/scripts/controllers/build.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ angular.module('openshiftConsole')
     var watches = [];
 
     var setLogVars = function(build) {
-      $scope.logCanRun = !(_.includes(['New', 'Pending', 'Error'], build.status.phase));
+      $scope.logCanRun = !(_.includes(['New', 'Pending', 'Error'], _.get(build, 'status.phase')));
```
Comment (spadgett) [inline app/scripts/controllers/build.js]: I'm confused how `buildResolved` is called with a null value? Just worried the problem runs deeper.

Comment (benjaminapetersen) [reply app/scripts/controllers/build.js]: I noticed it due to a timeout with an API response (I think).

Comment (spadgett) [reply app/scripts/controllers/build.js]: Shouldn't that have gone to the `buildRejected` callback?

Comment (benjaminapetersen) [reply app/scripts/controllers/build.js]: That would make sense.   Hmm. Not sure the cause then.

**Pull request closed without merging**